extras/tools/comparison.py

```diff
@@ -212,7 +212,7 @@ def compile_pvdata(summary, pvdata, matchsc):
         sc = elem.get('spec', {}).get('storageClassName', '')
         if matchsc and sc not in matchsc:
             sys.stderr.write(
-                'ignoring: {} from storage class "{}"\n'.format(g["path"], sc))
+                'ignoring: {} from storage class "{}"\n'.format(g["path"] if g else ma['glusterBlockShare'], sc))
```

Comment:
We probably need to catch KeyError exceptions (or use .get) instead of using an if statement here. And if we are adding fallbacks we should fall back to something really expected like name of the pv.